openreview/venue_request/process/matching_pre_process.py

```diff
@@ -1,4 +1,20 @@
 def process(client, note, invitation):
+    _, baseurl_v2 = openreview.tools.get_base_urls(client)
+    client_v2 = openreview.api.OpenReviewClient(baseurl=baseurl_v2, token=client.token)
+    forum_note = client.get_note(note.forum)
+    venue_id = forum_note.content.get('venue_id', '')
+    domain = client_v2.get_group(venue_id)
+    submission_venue_id = domain.content['submission_venue_id']['value']
+    senior_area_chairs_name = domain.content.get('senior_area_chairs_name', {}).get('value')
+    compute_affinity_scores = note.content.get('compute_affinity_scores') == 'Yes'
```

Comment:
can you check the value `No` instead? we are going to add more options that represent the affinity model but "No" will stay: https://github.com/openreview/openreview-py/pull/1843/files